pkg/vm/context/cairo_run.go

```diff
@@ -0,0 +1,49 @@
+package context
+
+import (
+	"github.com/lambdaclass/cairo-vm.go/pkg/parser"
+	"github.com/lambdaclass/cairo-vm.go/pkg/runners"
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm"
+)
+
+type RunResources struct {
+	NSteps *uint
+}
+
+type BuiltinHintProcessor struct {
+	//Todo: ExtraHints update to map[string][dyn func]
+	ExtraHints   map[string]string
+	RunResources *RunResources
+}
+
+type CairoRunConfig struct {
+	Entrypoint   string
+	TraceEnabled bool
+	RelocateMem  bool
+	Layout       string
+	ProofMode    bool
+	SecureRun    *bool
+}
```

Comment:
We can probably reduce this to 
```suggestion
type CairoRunConfig struct {
	TraceFile *string
	MemoryFile * string
}
```
Given our current functionality

Reply:
Done